rdrf/rdrf/settings.py

```diff
@@ -585,7 +585,7 @@
 
 # In case you set up customised email templates for the "new-patient" notification, you should
```

Comment:
Can we update the comment as well?
It should say that this turns off django-registration-redux's email notifications as we would like to send email registration through trrf's email notification system.

Reply:
Updated.